modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/portlet/action/EditEntryMVCActionCommand.java

```diff
@@ -602,7 +629,7 @@ private BlogsEntry _updateEntry(ActionRequest actionRequest)
 
 		if (Validator.isNotNull(portletResource)) {
 			MultiSessionMessages.add(
-				actionRequest, portletResource + "requestProcessed");
+				actionRequest, portletResource + "_requestProcessed");
```

Comment:
Why cannot we use the old "requestProcessed"? Maybe I forgot something, but shouldn't it work fine with it as long as we keep the `liferay-ui:success` in the jsps?